_inc/search/components/api.js

```diff
@@ -0,0 +1,14 @@
+class JetpackSearchAPI {
+	getAPIUrl( query ) {
+		let url = 'https://public-api.wordpress.com/rest/v1.3/sites/20115252/search?q=';
+		url += encodeURIComponent( query );
+		url += '&fields=author';
+		return url;
+	}
+
+	fetch( query ) {
+		return window.fetch( this.getAPIUrl( query ) );
```

Comment:
See convo about using `apiFetch`/`fetch` and what you need to account for when bringing this code to .com:

https://github.com/Automattic/jetpack/pull/12823#pullrequestreview-256217084

import { addQueryArgs } from '@wordpress/url';

Reply:
Let me try summarizing what the conversation linked above says:

1. We can't use `@wordpress/api-fetch` because it always sets a `X-WP-Nonce` header which WordPress.com disallows. 
2. If we use [`fetch`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API), we avoid the `X-WP-Nonce` header issue.

Is this correct?